utils/lit/lit/TestingConfig.py

```diff
@@ -1,13 +1,39 @@
 import os
 import sys
+import platform
 
 OldPy = sys.version_info[0] == 2 and sys.version_info[1] < 7
 
+def strip_dxil_validator_path(env_path):
```

Comment:
Looks fine, but in case you're curious, the short version could look like:
```py
def strip_dxil_validator_path(env_path):
    dxil_name, separator = ('dxil.dll', ';') if platform.system() == 'Windows' else ('dxil.so', ';')
    return separator.join([
        p for p in env_path.split(separator)
        if not os.path.isfile(os.path.join(p, dxil_name))
        ])
```

Reply:
This is much better.